packages/marshal/src/marshal-justin.js

```diff
@@ -445,3 +451,60 @@ const decodeToJustin = (encoding, shouldIndent = false, slots = []) => {
 };
 harden(decodeToJustin);
 export { decodeToJustin };
+
+/**
+ * @param {Passable} passable
+ * @param {boolean} [shouldIndent]
+ * @returns {string}
+ */
+export const passableAsJustin = (passable, shouldIndent = true) => {
+  let slotCount = 0;
+  // eslint-disable-next-line no-plusplus
+  const convertValToSlot = val => `s${slotCount++}`;
```

Comment:
consider:

```suggestion
  const convertValToSlot = val => `s${(slotCount += 1)}`;
```

not exactly the same... using `+=` is more like `++slotCount`. But close enough?

Reply:
All else being equal, I would prefer that. But I'd to start counting at zero, because that would reflect what the slot indexes would be. And I don't want to initialize to `-1` because that just moves the surprise elsewhere that's not better.

Reply:
Resolving. But if you object, please unresolve.

Reply:
I don't feel all that strongly about it, but... what's the justification for this exception to `no-plusplus`? I suggest including it in the comment.

Initializing to `-1` doesn't seem that surprising to me.

Reply:
I merged this PR before adding the requested comment. #2801 would correct.